package.json

```diff
@@ -37,34 +39,32 @@
     "@babel/core": "^7.24.7",
     "@babel/eslint-parser": "^7.19.1",
     "@babel/node": "^7.22.5",
+    "@babel/plugin-transform-react-jsx": "^7.20.7",
     "@babel/preset-env": "^7.19.4",
     "@babel/preset-flow": "^7.18.6",
-    "babel-jest": "^29.2.1",
+    "@babel/register": "^7.24.6",
+    "babel-plugin-module-resolver": "^5.0.2",
     "commander": "^9.4.1",
-    "eslint": "^8.26.0",
-    "eslint-config-airbnb-base": "^15.0.0",
+    "deep-equal-json": "^1.0.0",
```

Comment:
aria-query is definitely going to end up with zero deps, but since the replacement function isn't exposed (intentionally), i can either use the dev dep, or i can duplicate the code, and the dev dep felt simpler.